pkg/snapshot/snapshotter/snapshotter_test.go

```diff
@@ -1115,3 +1283,61 @@ func prepareStoreWithDeltaSnapshots(storeContainer string, numDeltaSnapshots int
 
 	return store
 }
+
+// addObjectsToStore adds objects to the given store. It creates and saves objects based on the provided parameters.
+// The objectType can be either "Chunk" or "Composite". The kind specifies the type of the snapshot Full/delta
+// The startRevision and lastRevision define the revision range of the objects.
+// The numOfSnapshots determines the number of objects to be created.
+// The creationTime specifies the creation time of the object
+func addObjectsToStore(store brtypes.SnapStore, objectType string, kind string, startRevision int, lastRevision int, numOfSnapshots int, creationTime time.Time) {
```

Comment:
In IMO change the func singnature to return error
```go
func addObjectsToStore() error{
...
}
```
Let the caller handle handle the error with like this `Expect(err).NotTo(HaveOccurred())`